### ISSUE TITLE
No ARM64 support

### ISSUE THREAD

**Issue by Gozzeh** [anthonix/ffts#34]
The code will not compile for apple devices that support 64-bit which is now a requirement.

64-bit support would be great.

**Comment by fsausset** [comment]
Indeed, I'm eagerly waiting this 64-bits support, as I don't have the skills to implement it by myself.

**Comment by bitbank2** [comment]
If no one has already done this, I can do it. There are also some 32-bit optimization opportunities that I would like to add. Please let me know if there isn't already an ARM64 fork and I'll work on it this week.

**Comment by linkotec** [comment]
That would be great improvement! I have been planning to implement AArch64 Neon optimization but other projects take time.. Looking forward for your fork.

**Comment by bitbank2** [comment]
I forked it and started updating the code to work on aarch64. Some of the inline asm can be tossed because intrinsics actually compile to good code on iOS and Linaro Linux. The .S files may not be needed either. I did some perf testing of ARM32 versus ARM64 (my gcc_perf project) and the Linaro aarch64 version of GCC is much harder to beat with hand-written asm than the older version of GCC. I'm not sure how long this will take, but I'm not a patient person :). First priority for me is to get it working on aarch64 Linux (my Dragonboard410c), then iOS and finally Android.

**Comment by jaadelgren** [comment]
Any updates on this? Lincotec, I saw your fork and thought about using that from now on, but I just spent almost two days getting a build working for iOS using the old autoconf tools so I didn't really want to (potentially) start over. Also, did you add in other optimizations? I thought I saw some new stuff sprinkled throughout :p For now I've just been disabling NEON, but let me know if you guys make some headway. Much appreciated!
